cli/azd/pkg/project/scaffold_gen_test.go

```diff
@@ -74,3 +74,70 @@ func Test_genBicepParamsFromEnvSubst(t *testing.T) {
 		})
 	}
 }
+
+func Test_DependentResourcesOf(t *testing.T) {
```

Comment:
Out of curiosity, do you still find value in this test after the recent changes? I understand this as an expectation test, and could easily lean towards removing it altogether given the implementation being generally *trivial*

Reply:
Yeah I had the same thought - I can remove it